ClemBot.Bot/bot/services/message_handling_service.py

```diff
@@ -90,6 +91,11 @@ async def on_guild_message_received(self, message: discord.Message) -> None:
         # Enabled embed message links
         await self.handle_message_links(message)
 
+        # Check if author is bot and is in our allowed bot input ids
+        if message.author.bot and message.author.id in bot_secrets.secrets.allow_bot_input_ids:
+            ctx = await self.bot.get_context(message)
+            await self.bot.invoke(ctx)
+
         # Primary entry point for handling commands
```

Comment:
This is invoking a command twice

Reply:
The user either is or isn't a bot, and process_command will return if the user is a bot, so it'll only run once.